test/CoinbaseSmartWalletFactory.t.sol

```diff
@@ -32,6 +33,20 @@ contract CoinbaseSmartWalletFactoryTest is Test {
         factory.createAccount{value: 1e18}(owners, 0);
     }
 
+    function test_exitIfAccountIsAlreadyInitialized() public {
+        CoinbaseSmartWallet a = factory.createAccount{value: 1e18}(owners, 0);
+        vm.expectCall(address(a), abi.encodeCall(CoinbaseSmartWallet.initialize, (owners)), 0);
+        CoinbaseSmartWallet a2 = factory.createAccount(owners, 0);
+        assertEq(address(a), address(a2));
+    }
+
+    function testRevertsIfLength32ButLargerThanAddress() public {
```

Comment:
our test function names are not consistent right now, but let's at least try to pick one format when adding new ones